engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -865,6 +876,10 @@ public void saveExtraDhcpOptions(final String networkUuid, final Long nicId, fin
             configureNicProfileBasedOnRequestedIp(requested, profile, network);
         }
 
+        if(Objects.isNull(profile.getMtu())) {
```

Comment:
Same applies here. Please use the Apache Commons validator

Reply:
changed, please check again